projects/common-components/src/lib/dialog/dialog.component.scss

```diff
@@ -0,0 +1,75 @@
+:host {
+    width: 100%;
+    height: 100vh;
+    background: #04050936;
+	position: fixed;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+%buttons {
+	padding: 15px;
+	font-size: 1.2em;
+	border: none;
+	border-radius: 5px;
+	color: white;
+	cursor: pointer;
+	&:hover {
+		opacity: 0.8;
+	}
+}
+.dialog {
+	width: 400px;
+	height: 300px;
+	background: white;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	border-radius: 10px;
+	padding: 20px 30px;
+	margin: auto;
+	&__header {
+		display: flex;
+		justify-content: space-between;
+	}
+	&__separator {
+		margin: auto;
+		border: none;
+		border-top: 1px solid #d4d4d4;
+		width: 100%;
+		margin: 10px 0;
+		flex-grow: 1;
+	}
+	&__title {
+		color: #333;
+		font-size: 2em;
+	}
+	&__close {
+		color: #888888;
+		border: none;
+		height: 30px;
+		font-size: 2em;
+		cursor: pointer;
+		&:hover {
+			font-size: 2.3em;
+			transition: 0.2s;
+		}
+	}
+	&__message {
+		color: #5d5d5d;
+		font-size: 1.3em;
+		flex-grow: 2;
+	}
+	&__buttons {
+		display: flex;
+		justify-content: space-between;
+	}
+	&__yes-button {
+		@extend %buttons;
+		background: #328eff;
+	}
+	&__no-button {
+		@extend %buttons;
+		background: #ef4e1d;
+	}
```

Comment:
pls define yes and no as modifiers for button element

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/69210975/223937151-1a5bee74-f535-4b42-80d4-b84dcaec9caa.png)

Please use nesting for modifiers

Reply:
ok, done

Reply:
&__button {
  &_yes {}
  
  &_no {}
}

Reply:
here you duplicate class defining 
![image](https://user-images.githubusercontent.com/69210975/224636081-11128b8a-4c0e-4f17-9b07-2d24e917bcd4.png)


&__button {
    &_yes {}
    &_no {}
}